namesilo.php

```diff
@@ -635,49 +646,126 @@ public function editPackage($package, array $vars = null)
      */
     public function manageModule($module, array &$vars)
     {
-        // Load the view into this object, so helpers can be automatically added to the view
-        $this->view = new View('manage', 'default');
-        $this->view->base_uri = $this->base_uri;
-        $this->view->setDefaultView(self::$defaultModuleView);
+        $action = isset($_GET['action']) ? $_GET['action'] : null;
 
-        #
-        #
-        # TODO: add tab to check status of all transfers: check if possible with Namesilo...
-        # ref: NamesiloDomainsTransfer->getList()
-        #
-        #
-        $link_buttons = [];
-        foreach ($module->rows as $row) {
-            if (isset($row->meta->key)) {
-                $link_buttons[] = [
-                    'name' => Language::_('Namesilo.manage.sync_renew_dates', true),
-                    'attributes' => [
-                        'href' => [
-                            'href' => $this->base_uri . 'settings/company/modules/addrow/'
-                                . $module->id . '?action=sync_renew_dates'
+        if ($action == 'create_packages') {
+            // Load Namesilo packages
+            Loader::load(__DIR__ . DS . 'includes' . DS . 'namesilo_packages.php');
+            $this->NamesiloPackages = new NamesiloPackages();
+
+            // Fetch stored packages vars, if available
+            $module_row = $this->getRow();
+            $packages_vars = $this->NamesiloPackages->getPackagesVars($module_row->id);
+
+            $vars = [];
```

Comment:
If I don't empty $vars, the error `Undefined offset: 0 on line 462 in ./app/models/module_manager.php` is thrown. 

I'm not sure why this happens, but I guess the ModuleManager::setMeta() function expects a numerically indexed array with the meta data on $vars, but inside this function $vars contains the post submitted data in an array of key/value pairs, so it can't find the 0 element and throws the error.

That's why I used $_POST because $vars would remain empty for the rest of the execution from this point. To avoid this error and not use $_POST anymore, I used this alternative:
```
$post = $vars;
$vars = [];
```
Let me know if you know a better approach or have a suggestion.